crates/bevy_image/src/image.rs

```diff
@@ -851,6 +851,8 @@ impl Image {
 
     /// Resizes the image to the new size, by removing information or appending 0 to the `data`.
     /// Does not properly scale the contents of the image.
+    ///
+    /// If you need to keep pixel data intact, use [`Image::resize_in_place_2d`].
```

Comment:
Maybe `resize` should take a `fill` param too?

Reply:
Removed